tests/ceph_installer/test_ansible.py

```diff
@@ -119,7 +119,16 @@ def run(ceph_cluster, **kw):
     if test_data.get("luns_setting", None) and test_data.get("initiator_setting", None):
         ceph_installer.add_iscsi_settings(test_data)
 
-    LOG.info("Ceph ansible version " + ceph_installer.get_installed_ceph_versions())
+    # Check if ansilbe.log file has any logs initially
+    if custom_log:
+        ansible_log_file, _ = ceph_installer.exec_command(
+            cmd="cat {}/ansible.cfg | grep log_path | cut -d \" \" -f3 | tr -d '\n\r'".format(
```

Comment:
It is recommended to use format strings i.e. `cmd=f"cat {ansible_dir}/ansible.cfg | ... "`

Reply:
Sure, will update the changes.

Reply:
Updated the changes.